content/en/containers/kubernetes/installation.md

```diff
@@ -177,7 +177,7 @@ agent:
         - <GROUP_ID>
 {{< /highlight >}}
 
-- Replace `<USER_ID>` with the UID to run the Datadog Agent.
+- Replace `<USER_ID>` with the UID to run the Datadog Agent. [Datadog recommends setting this value to 100 since Datadog Agent v7.48+][26].
 - Replace `<GROUP_ID>` with the group ID that owns the Docker or containerd socket.
```

Comment:
```suggestion
- Replace `<GROUP_ID>` with the group ID that owns the Docker or containerd socket.

[1]: https://github.com/DataDog/documentation/blob/63b3271a3abda3ba035d3773fba3dc07ec97c344/content/en/data_security/kubernetes.md?plain=1#L50-L51
```

Reply:
I want to use this link instead of the GitHub url.
https://docs.datadoghq.com/data_security/kubernetes/#running-container-as-root-user

Reply:
updated!